README.md

```diff
@@ -80,10 +80,11 @@ Starlette does not have any hard dependencies, but the following are optional:
 
 * [`requests`][requests] - Required if you want to use the `TestClient`.
 * [`aiofiles`][aiofiles] - Required if you want to use `FileResponse` or `StaticFiles`.
+* [`jinja2`][jinja2] - Required if you want to use the default template configuration.
```

Comment:
The link for jinja2 is missing. @tomchristie 

![image](https://user-images.githubusercontent.com/9675939/48212735-33a87b00-e3b7-11e8-9da0-6258994194a6.png)

Reply:
Thanks!